java/src/org/openqa/selenium/grid/node/config/NodeOptions.java

```diff
@@ -18,50 +18,35 @@
 package org.openqa.selenium.grid.node.config;
 
 import com.google.common.annotations.VisibleForTesting;
-import com.google.common.collect.HashMultimap;
-import com.google.common.collect.ImmutableMap;
-import com.google.common.collect.ImmutableMultimap;
-import com.google.common.collect.ImmutableSet;
-import com.google.common.collect.Multimap;
+import com.google.common.collect.*;
+import org.openqa.selenium.Platform;
+import org.openqa.selenium.*;
+import org.openqa.selenium.grid.config.Config;
+import org.openqa.selenium.grid.config.ConfigException;
+import org.openqa.selenium.grid.node.Node;
+import org.openqa.selenium.grid.node.SessionFactory;
+import org.openqa.selenium.internal.Require;
+import org.openqa.selenium.json.Json;
+import org.openqa.selenium.json.JsonOutput;
+import org.openqa.selenium.net.NetworkUtils;
+import org.openqa.selenium.net.Urls;
+import org.openqa.selenium.remote.Browser;
+import org.openqa.selenium.remote.service.DriverService;
+
 import java.io.File;
 import java.lang.reflect.Method;
 import java.lang.reflect.Modifier;
 import java.net.URI;
 import java.net.URISyntaxException;
 import java.time.Duration;
-import java.util.ArrayList;
-import java.util.Collection;
-import java.util.Comparator;
-import java.util.HashMap;
-import java.util.List;
-import java.util.Map;
-import java.util.Optional;
-import java.util.ServiceLoader;
+import java.util.*;
```

Comment:
I am not sure as to what the guidelines are in terms of using wild card imports vs using the actual classes. It would be good if we could revert it to explicitly importing the classes we use

Reply:
We don't allow wildcard imports. If you run `./scripts/format.sh`, you should get this sorted out

Reply:
I restored the original discrete imports, eliminating the wildcards and the unnecessary source deltas.